vars/kola.groovy

```diff
@@ -117,12 +117,19 @@ def call(params = [:]) {
             runKola(id, 'run', "--parallel=${parallel} ${args} ${extraArgs}")
         } else {
             // basic run
-            if (!params['skipBasicScenarios']) {
+            // Check if kola has old or new basic scenarios
+            def out = shwrapCapture("""
+            cd ${cosaDir}
+            cosa kola list
+        """)
+            def newBasic.nvmeExist = out.readLines().any { it =~ /basic\.nvme/ }
+
+            if (!params['skipBasicScenarios'] && !exists) {
                 id = marker == "" ? "kola-basic" : "kola-basic-${marker}"
                 ids += id
                 def skipSecureBootArg = ""
-                if (params['skipSecureBoot']) {
-                    skipSecureBootArg = "--skip-secure-boot"
+            if (params['skipSecureBoot'] && !exists) {
```

Comment:
```suggestion
                if (params['skipSecureBoot']) {
```

We don't need to check `exists` again (suggest renaming that to e.g. `basicScenariosBuiltIn` or something more descriptive).